README.md

```diff
@@ -1,393 +1,296 @@
-The default is 2 minutes. This timeout might, for example, occurr when the given file doesn't exist the
-browser didn't start or the localhost tunnel isn't running.
+### General Settings
 
-__`delay` *{Integer}*__<br />
-Multiple test files will be run in sequence on the same browser. This option sets the delay (in ms) between
-launching the browser again with the same file.
+#### `delay`
+Type: `Number`  
+Default value: `1000`  
+The delay (in milliseconds) between multiple test pages within a single browser.
 
-### Reporters
+#### `port`
+Type: `Number`  
+Default value: `3996`  
+The port of the static fileserver used to serve the tests. This will also be used by [Localhost tunneling services](#localhost-tunneling).
 
-The `reporter` option allows you to use all of the console reporters included in the
-[Mocha testing library](https://mochajs.org/#reporters).
+#### `reporter`
+Type: `String`  
+Default value: `'dot'`  
+See [Mocha reporters](https://mochajs.org/#reporters).
 
+#### `root`
+Type: `String`
+Default value: `process.cwd()`  
+The root path (or base URL) of the static fileserver used to serve the tests. Any test file will be relative to this path.
 
-### Launching browsers
+#### `timeout`
+Type: `Number`  
+Default value: `120`  
+The time (in seconds) to wait for a test page to report back before an error is thrown. This timeout might, for example, occurr when the given file doesn't exist, the browser didn't start, or the localhost tunnel wasn't running.
 
-The `launch` and `browsers` options are used to set the environment and the browsers you want
-to start. [Launchpad](https://github.com/ekryski/launchpad) is the browser launcher library used
-by Testee which allows you to start most locally installed browsers as well as Browserstack workers
-and even browsers on remote systems running the [Launchpad Server](https://github.com/ekryski/launchpad#the-launchpad-server).
 
-The most common case will be launching a local browser (which is also the default) with no settings:
+### Browser Settings
 
-```js
-{
-  "browsers" : [ "firefox", "safari" ]
-}
-```
+**Note:** Depending on your OS, the target browsers should not already be open/running before using this library. Electron and PhantomJS are the exceptions, as they can always be started multiple times.
 
-#### Browserstack
+#### `browsers`
+Type: `Array`  
+Default value: `['phantom']`  
+The browsers that will be used to run tests. For local browsers, use a browser name string (see [launchpad](https://npmjs.com/launchpad) for more info). For a remote/BrowserStack browser, use a [browser object](https://npmjs.com/browserstack#browser-objects). Mixing both types is not currently possible.
 
-Browserstack hosts virtual machines running specific versions of web browsers. It is an extremely useful
-tool for cross-browser testing running a remote desktop in you browser. To use BrowserStack via the configuration
-API you need to provide a username and password:
```

Comment:
Some paragraphs that more thoroughly explain the examples were removed, like this one. Can they be added back please?